### PR TITLE
Practice app user update

### PR DESCRIPTION
**Pull request by meltemarsl**
Please Make a code review and merge if there is not any problem. If there are sone things that must change please write on the linked issue about UpdateUser Endpoint.

Please control these criteria.(If you want you can add extra criteria.)
- [x] Documentation
- [x] Readability
- [x] Code works correctly or not?
- [x] Tests

### CODE REVIEW

**Comment by ahmetdadak** [comment]
Nicely documented well-written code! Nice job.

**Comment by darktheorys** [comment]
I also resolved conflicts that can prevent merging, you can view them.

File: practice-app/platon_api/rest_api/update_user/update_user.py [hunk 1]
```diff
@@ -95,6 +99,8 @@ def updateUser(request):
             if(len(job_name) != 0):
                 user.job_name = job_name
                 user.save()
-            return HttpResponse("ifff")
+            #If program reaches at this point, returns confirmation message.
+            return HttpResponse("User information is updated!")
         except:
-            return HttpResponse("Error")    
+            #If program fails, it returns this message.
+            return HttpResponse("Error! Information could not be changed.")
```
Comment (darktheorys) [inline practice-app/platon_api/rest_api/update_user/update_user.py]: There should be get request handling.

**Comment by darktheorys** [comment]
I have discovered that your api gives error on get requests. You should handle it i think. Also, because none of us have found it until now. I think you may want to allow only post requests. I wrote a comment where you can solve it.

**Pull request merged**